frontend/src/components/Header/Header.js

```diff
@@ -1,26 +1,29 @@
 import React from 'react';
-import {
-  BrowserRouter as Router,
-  Link
-} from "react-router-dom";
+import { Link } from 'react-router-dom';
```

Comment:
Should we be using `NavLink` here instead? That will allow us to  set activeStyle or somefn. This could easily be a separate issue but I'll let you choose.

Reply:
Fixed in the latest push